bundles/org.eclipse.equinox.servletbridge/src/org/eclipse/equinox/servletbridge/FrameworkLauncher.java

```diff
@@ -29,17 +29,11 @@
 import javax.servlet.ServletContext;
 
 /**
- * The FrameworkLauncher provides the logic to:
- * 1) init
- * 2) deploy
- * 3) start
- * 4) stop
- * 5) undeploy
- * 6) destroy
- * an instance of the OSGi framework.
- * These 6 methods are provided to help manage the life-cycle and are called from outside this
- * class by the BridgeServlet. To create an extended FrameworkLauncher over-ride these methods to allow
- * custom behavior.
+ * The FrameworkLauncher provides the logic to: 1) init 2) deploy 3) start 4)
+ * stop 5) undeploy 6) destroy an instance of the OSGi framework. These 6
```

Comment:
This breaks the comment as it's done manually, rather than via using ol tag. Please change accordingly.